test/django/test_email_backend.py

```diff
@@ -172,4 +172,10 @@ def test_settings_options():
 
     with mock.patch.object(Transmissions, 'send'):
         mailer(get_params())
-        Transmissions.send.assert_called_with(**SPARKPOST_OPTIONS)
+        expected_kargs = dict(get_params())
```

Comment:
I don't think we need a deep copy here, but I think `get_params().copy()` would be more explicit.